Sources/Sentry/SentrySdkInfo.m

```diff
@@ -0,0 +1,87 @@
+#import "SentrySdkInfo.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation SentrySdkInfo
+
+- (instancetype)initWithSdkName:(NSString *)sdkName andVersionString:(NSString *)versionString
+{
+    if (self = [super init]) {
+        _sdkName = sdkName;
```

Comment:
H: if `sdkName` is null or empty, it should return a nullable `SentrySdkInfo` as it really doesn't matter the version of it

Reply:
I wouldn't add this in here. If the `sdkName` and/or `versionString` are nil and we don't want to send the `SentrySdkInterface` with the envelope, the envelope should take care of that.

Reply:
ok addressed here then https://github.com/getsentry/sentry-cocoa/pull/626/files#r456382544